src/gps/dateTime.c

```diff
@@ -133,3 +133,118 @@ tiny_millis_t getUptime() {
 int getUptimeAsInt() {
    return (int) getUptime();
 }
+
+void getDateTimeFromEpochMillis(DateTime *dateTime, millis_t millis) {
```

Comment:
I don't necessarily agree with the readability tradeoff vs the minor performance gain here.  Either way space needs to be made on the stack and values copied.  GCC does a very good job of optimizing this and considering the speed of the processor even on the MK1 device the savings are a no-op.  Also its easier for someone to mess up programming this method by accidentally passing a NULL pointer to the first argument.  You don't have that issue when you return the struct.  But that said this is a fundamental difference in programming style and it doesn't matter too much.